terraform-plans/configs/hardware-observer-operator_main.tfvars

```diff
@@ -42,7 +42,7 @@ templates = {
     destination = ".github/.jira_sync_config.yaml"
     vars = {
       component = "hardware-observer",
-      epic_key  = "SOLENG-190"
+      epic_key  = "SOLENG-1100"
```

Comment:
Did you change by mistake? I think we can leave the epic as hardware observer

Reply:
Oh good catch thanks, yep that's a mistake.